content/20.treatments.md

```diff
@@ -302,8 +302,18 @@ All of these factors could potentially influence treatment outcome.
 Furthermore, the authors acknowledge that the effect of the drugs might be different in females and pediatric subjects, since these subjects were not the part of the study.
 
 **Summary.**
-In vitro evidence shows HCQ may be an effective therapeutic against SARS-CoV-2 and COVID-19.
-Multiple clinical studies have already been carried out to assess this possibility.
+In vitro evidence shows HCQ may be an effective therapeutic against SARS-CoV-2 and COVID-19. 
+It was recently discussed (@doi:10.1093/jac/dkaa114) that since EC90 of CQ in Vero E6 cells (6.90 μM) could be reached and well endured in rheumatoid arthritis patients, it could be as well be administered to COVID-19 patients.
```

Comment:
```suggestion
It was recently discussed [@doi:10.1093/jac/dkaa114] that since EC90 of CQ in Vero E6 cells (6.90 μM) could be reached and well endured in rheumatoid arthritis patients, it could also be administered to COVID-19 patients.
```
I'm not sure what "reached" means in this context